spec/models/ont/flowcell_spec.rb

```diff
@@ -62,4 +72,53 @@
       expect(run.flowcells.count).to be(initial_flowcell_count - 1)
     end
   end
+
+  context 'flowcell adressing' do
+    it 'returns coordinates for PromethION' do
+      run = create(:ont_promethion_run, flowcell_count: 24)
+      first = run.flowcells.find_by(position: 1)
+      last = run.flowcells.find_by(position: 24)
+
+      expect(first.position).to eq(1)
+      expect(first.device_id).to eq('1A')
+
+      expect(last.position).to eq(24)
+      expect(last.device_id).to eq('3H')
```

Comment:
Thank you 👍 maybe it could be called `coordinate` instead of `device_id`? the `_id` might be confusing

Reply:
I have already renamed it to position_display before this suggestion. I think, singular and plural for coordinate is confusing. The method can also return a number (position) if there is no mapping for a device as well.